diagnostic.md

```diff
@@ -36,13 +43,20 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --request GET http://www.example.com/donuts/maplebacon
 ```
 
 Write an AJAX request to retrieve a single donut from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = function(){
+  $.ajax({
+   url: 'http://www.example.com/donuts/maplebacon',
```

Comment:
This could be right - in this case, you'd be using `maplebacon` to represent one specific donut on your back-end. What's more likely, though, is that you'd represent that specific donut with either a number (e.g. `donuts/3`) or an alphanumberic string (e.g. `donuts/987h87yd`).